packages/react-components/react-message-bar/library/src/components/MessageBarBody/useMessageBarBodyStyles.styles.ts

```diff
@@ -11,6 +11,9 @@ const useRootBaseStyles = makeResetStyles({
   ...typographyStyles.body1,
   gridArea: 'body',
   paddingRight: tokens.spacingHorizontalM,
+  '& .fui-Link, & a': {
```

Comment:
Instead of the .fui-Link hardcoded here, that can be imported from the `@fluentui/react-link` package and being used here.